src/main/kotlin/org/jitsi/nlj/rtp/codec/vp8/Vp8Packet.kt

```diff
@@ -67,6 +67,14 @@ class Vp8Packet private constructor (
         /** This uses [get] rather than initialization because [isMarked] is a var. */
         get() = isMarked
 
+    val hasTemporalLayerIndex = DePacketizer.VP8PayloadDescriptor.hasTemporalLayerIndex(buffer, payloadOffset, payloadLength)
```

Comment:
This uses the `buffer` constructor argument instead of `this.buffer`. I see most of the existing fields do the same, but `isKeyframe` uses `this.buffer`. I think they should all use `this.buffer` to avoid something similar to the RTX issue, though I don't think this is causing any problems right now.

Reply:
Actually this only makes a difference for the setters of `var`s.